src/AzureExtension/Strings/en-US/Resources.resw

```diff
@@ -726,4 +726,9 @@
     <value>Rejected</value>
     <comment>Shown in Toast Notification, title line</comment>
   </data>
+  <data name="Widget_Template.LoadingFindingPullRequests" xml:space="preserve">
+    <value>Finding your pull requests.

```

Comment:
Why is a newline here?

Reply:
I did it becuase it looks better in the widget UX with line wrapping.  However, I can see how it would cause a problem in localization. I've removed it and can adjust that in the template if necessary to tweak the presentation.